[16.0][mig] customer_wallet_(account-portal-pos) (#397)
coopiteasy/addons: customer_wallet_portal/readme/DESCRIPTION.rst

> My Home displays customer wallet amount and details.

Reviewer: the formulation is a little weird. i think this would be better:
```suggestion
Display customer wallet amount and details on the customer portal.
```
